src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -364,6 +365,40 @@ public void onBlockDropItems(BlockDropItemEvent e) {
                     }
                 }
             }
+
+            // We also want to double crops
```

Comment:
comment could be clearer
```suggestion
            // Handle double drops for Farmer Talisman
```